uvicorn/main.py

```diff
@@ -282,6 +282,12 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     default=None,
     help="Maximum number of seconds to wait for graceful shutdown.",
 )
+@click.option(
+    "--timeout-worker-is-alive",
+    type=float,
+    default=5,
```

Comment:
I'm not sure this is wanted as default value?

Reply:
Ah, I've mentioned before: https://github.com/encode/uvicorn/pull/2397#issuecomment-2353586324

Reply:
Yes, it's a good default value for most programs. Only those with heavy loads require larger values.

Reply:
Additionally, the default timeout for a single request in httpx is also 5 seconds.

Reply:
Some checks failed. Can I fix it?

Reply:
Yes, please fix it.

Reply:
There's an unrelated issue on the pipeline.

Reply:
Thank you. I'll.
Excuse me, I'm not familiar with the forked repository contribution.
I think, I need to fork `socrateslee:feature/timeout_worker_is_alive`, but I'm not sure how to do it.
Could you please explain for me?

Reply:
> Yes, it's a good default value for most programs. Only those with heavy loads require larger values.

Well... Those with heavy loads will have to change this. Which is a bothersome for them.